uikit/views/src/main/res/layout/layout_search_result_list_internal.xml

```diff
@@ -27,28 +27,76 @@
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools">
 
-    <FrameLayout
+    <androidx.constraintlayout.widget.ConstraintLayout
         android:layout_width="match_parent"
         android:layout_height="match_parent">
 
+        <androidx.constraintlayout.widget.Guideline
+            android:id="@+id/topGuideline"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:orientation="horizontal"
+            app:layout_constraintGuide_percent="0.08" />
+
         <com.sygic.maps.uikit.views.common.EmptyRecyclerView
             android:id="@+id/searchResultListRecyclerView"
-            android:layout_width="match_parent"
-            android:layout_height="match_parent"
+            android:layout_width="0dp"
+            android:layout_height="0dp"
             android:scrollbars="vertical"
-            app:emptyView="@id/searchResultListEmptyView"
+            app:emptyView="@id/searchResultListErrorViewAnimator"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
             tools:listitem="@layout/layout_search_item_result_internal" />
 
-        <com.sygic.maps.uikit.views.common.AdvanceInfoView
-            android:id="@+id/searchResultListEmptyView"
-            android:layout_width="match_parent"
-            android:layout_height="match_parent"
-            app:imageDescription="@string/magnifying_glass"
-            app:imageSource="@drawable/ic_search"
-            app:suggestionText="@string/try_changing_your_search_criteria"
-            app:titleText="@string/no_results_found" />
+        <ViewAnimator
+            android:id="@+id/searchResultListErrorViewAnimator"
```

Comment:
kedze sa cely layout vola `layout_search_result_list_internal.xml` tieto idcka by som zjednodusil, nech tam uz nie je zbytocne `searchResultList` znovu. podla mna staci len `errorViewAnimator` A aj v kode mas metodu, ktora je len `setErrorViewSwitcherIndex`

Reply:
mno nevím nevím. metoda ano, ta se jmenuje jen `setErrorViewSwitcherIndex`, ale proto, že jí můžeš volat jen na SearchResultList třídě. Kdežto ty id jsou vidět globálně chtě nechtě a nerad bych pak někomu s něčím kolidoval, když si to dá do projektu a už tam náhodou bude nějaký svůj `errorViewAnimator ` id mít. Můžeme o tom dát řeč, ale takto za mě.

Reply:
co sa tyka id, tak tam duplicity nevadia pokial nie su v jednom xml za sebou (ono chyba to nie je, len by ti naslo iba prvy vyskyt). Ako mala sanca tam je, ze ten list bude v xml s nejakym inym view s takym id, ale podla mna minimalna a pouzivame asi aj ine generickejsie idcka

Reply:
snažím se všude používat specifické id namísto generických, ale může se stát, že mi to někde unikne ;)